apps/flows/src/app/transaction-details-flow/transaction-details/transaction-details.component.html

```diff
@@ -0,0 +1,18 @@
+<kirby-page [title]="transactions.text">
+  <kirby-page-content>
+    <h1>{{ transactions.amount.amount }}</h1>
```

Comment:
The page header already uses `<h1>` for the page title, so this and subsequent headings _should_ be shifted one level down `<h1>` ➡️ `<h2>`, `<h2>` ➡️ `<h3>` etc. to maintain a proper page structure and outline.

Unfortunately we haven't got anything in place to easily control heading levels across different components, so it's not a _required_ change.

Reply:
Updated